habitat-lab/habitat/tasks/rearrange/utils.py

```diff
@@ -741,8 +745,23 @@ def get_camera_transform(cur_articulated_agent) -> mn.Matrix4:
         cam_info.attached_link_id
     ).transformation
     # Get the camera offset transformation
-    offset_trans = mn.Matrix4.translation(cam_info.cam_offset_pos)
-    cam_trans = link_trans @ offset_trans @ cam_info.relative_transform
+    # offset_trans = mn.Matrix4.translation(cam_info.cam_offset_pos)
+    cam_offset_transform = None
+    if cam_info.cam_look_at_pos == mn.Vector3(0, 0, 0):
```

Comment:
Can you clarify or comment the two options here?
Is `cam_look_at_pos` in local space of the camera? Why not use look_at all the time? Is Vector3(0,0,0) a special case (like -1 for unused?)